lib/Table.php

```diff
@@ -395,7 +395,7 @@ private function get_meta_data()
 	{
 		// as more adapters are added probably want to do this a better way
 		// than using instanceof but gud enuff for now
-		$quote_name = !($this->conn instanceof PgsqlAdapter);
+		$quote_name = !($this->conn instanceof PgsqlAdapter || $this->conn instanceof RedshiftAdapter);
```

Comment:
now `RedshiftAdapter` extends `PgsqlAdapter`, `$this->conn instanceof PgsqlAdapter` would be sufficient here